pandas/util/mixins.py

```diff
@@ -0,0 +1,28 @@
+"""
+Mixins which can't go to core.common or core.base due to circular imports
+"""
```

Comment:
But letting `StringMethods` use `PandasDelegate` will not fix this, as `PandasDelegate` is in the same dir as `IndexOpsMixins`.

I could try moving `IndexOpsMixin` into `core/indexing.py` -> Series/index -> loads indexing for  `IndexOpsMixin` -> loads string for `StringMethods` -> loads base for `NoNewAttributesMixin`. But somehow, despite the name, it has little to do with indexing :-(